test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -791,7 +802,7 @@
          (t/format :iso-offset-date-time b)]))]]) ; b_dt_tz_text
 
 (deftest datetime-diff-time-zones-test
-  (mt/test-drivers (mt/normal-drivers-with-feature :datetime-diff)
+  (mt/test-drivers (filter mt/supports-timestamptz-type? (mt/normal-drivers-with-feature :datetime-diff))
```

Comment:
Change 5: drivers that don't have timestamptz type shouldn't be tested here either.